index.js

```diff
@@ -72,9 +72,11 @@ client.get(mentionsPath, params, function(err, mentions) {
                                     mentions[i2].user.id_str) {
 
                                     failed = true;
+                                } else {
```

Comment:
Instead of all this, I think it should just be `if (mentions[i2].in_reply_to_user_id_str !== mentions[i2].user.id_str)`. If that works for you, then that's definitely what it should be. Thanks.